app/code/Magento/InventoryDistanceBasedSourceSelection/etc/db_schema.xml

```diff
@@ -8,11 +8,11 @@
 <schema xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:noNamespaceSchemaLocation="urn:magento:framework:Setup/Declaration/Schema/etc/schema.xsd">
     <table name="inventory_geoname" resource="default" engine="innodb">
-        <column xsi:type="varchar" name="country_code" nullable="false" length="64"/>
-        <column xsi:type="varchar" name="postcode" nullable="false" length="64"/>
-        <column xsi:type="varchar" name="city" nullable="false" length="64"/>
-        <column xsi:type="varchar" name="region" nullable="false" length="64"/>
-        <column xsi:type="varchar" name="province" nullable="false" length="64"/>
+        <column xsi:type="varchar" name="country_code" nullable="false" length="2"/>
+        <column xsi:type="varchar" name="postcode" nullable="false" length="20"/>
+        <column xsi:type="varchar" name="city" nullable="false" length="180"/>
+        <column xsi:type="varchar" name="region" nullable="false" length="100"/>
+        <column xsi:type="varchar" name="province" nullable="false" length="20"/>
```

Comment:
I would avoid decreasing maximum length just in case if some customization out there could use these fields in a different manner or for different data. 

Let's just keep ones which are increased